packages/core/parcel-bundler/src/utils/config.js

```diff
@@ -10,7 +10,7 @@ const PARSERS = {
 const existsCache = new Map();
 
 async function resolve(filepath, filenames, root = path.parse(filepath).root) {
-  filepath = path.dirname(filepath);
+  filepath = path.dirname(await fs.realpath(filepath));
```

Comment:
In what cases are these not realpaths? As far as I know assets are already resolved to their realpath.
I might be mistaken.

Reply:
Unfortunately not! There aren't many places where `realpath` is used in the codebase. I couldn't find a place where `realpath` is used for that purpose, and the yarn-workspace test shows that it actually does not resolve into the real path :)

I did a lot of tracing, and I figured this might be the best place to resolve them into real path. But, I may be wrong. Nevertheless, I don't think this would cause any performance issue.

Reply:
Yeah it looks good, don't expect any real performance issues as configs get cached anyways.